src/mame/includes/bublbobl.h

```diff
@@ -2,45 +2,60 @@
 // copyright-holders:Chris Moore, Nicola Salmoria
 
 #include "cpu/m6805/m68705.h"
+#include "cpu/m6800/m6801.h"
+#include "cpu/z80/z80.h"
```

Comment:
The header doesn't directly use these, but both missb2.cpp and bublbobl.cpp do, both of which include said header.